src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private DateTimeFormatter formatter;
+
+    public SalaryInfo() {
+        formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    }
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate localDateFrom = LocalDate.parse(dateFrom, formatter);
+        LocalDate localDateTo = LocalDate.parse(dateTo, formatter);
+
+        StringBuilder reportStringBuilder = new StringBuilder("Report for period ");
+        reportStringBuilder.append(dateFrom).append(" - ").append(dateTo);
+        for (String name : names) {
+            int salary = 0;
+            for (int i = 0; i < data.length; i++) {
+                String[] splittedData = data[i].split(" ");
+                final String Date_Index = splittedData[0];
```

Comment:
that's not what I was talking about)
I meant to create global constant, for example `private static final int DATE_INDEX = 0;`